conf/test_build.config

```diff
@@ -0,0 +1,28 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/rnafusion -profile test_build,<docker/singularity> --outdir <OUTDIR> -stub
+
+----------------------------------------------------------------------------------------
+*/
+
+process {
+    resourceLimits = [
+        cpus: 4,
+        memory: '15.GB',
+        time: '1.h'
+    ]
+}
+
+params {
+    config_profile_name        = 'Test build references profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Input data
+    build_references             = true
```

Comment:
shouldn't we have the `--no_cosmic` flag in `test_build`?

Reply:
Wasn't sure what could be considered "standard" for running the test profile, but I think you are right, i'll add it